engine/src/state_chain_observer/client/mod.rs

```diff
@@ -779,7 +787,10 @@ impl SignedExtrinsicClientBuilderTrait for SignedExtrinsicClientBuilder {
 								)],
 							),
 							&subxt_signer,
-						)
+							current_nonce,
+							Default::default(),
```

Comment:
We probably should set a mortality here

Reply:
The SCC uses 128 blocks

Reply:
Will create an issue

Reply:
https://linear.app/chainflip/issue/PRO-1130/add-mortality-to-cfe-version-extrinsic-submission-on-cfe-startup